### PR TITLE
Bundle size improvments by replacing some big methods of lodash

### PR DESCRIPTION
**Pull request by Faradey27**

### CODE REVIEW

File: src/GoogleAddressInput/google2address.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /*eslint camelcase: off*/
-import includes from 'lodash/includes';
-import omitBy from 'lodash/omitBy';
-import isUndefined from 'lodash/isUndefined';
+import isundefined from 'lodash/isUndefined';
+export const includes = (arr, value) => {
+  return arr && arr.find(item => item === value); // we compare only primitives
```
Comment (3LOK) [inline src/GoogleAddressInput/google2address.js]: suggestion: return !! to make sure a boolean is returned and not null.

**Comment by nirhart** [comment]
@Faradey27 can you please take a look at the conflict?

File: src/GoogleAddressInput/google2address.js [hunk 2]
```diff
@@ -12,20 +13,27 @@ export function google2address(google) {
...
-
-  return omitBy({
+  const result = {
     formatted: google.formatted_address,
     latLng: {
       lat: google.geometry.location.lat(),
       lng: google.geometry.location.lng()
     },
-    approximate: (!includes(google.types, 'street_address') && (!includes(google.types, 'premise'))),
+    approximate: (!(google.types || []).includes('street_address') && (!(google.types || []).includes('premise'))),
```
Comment (3LOK) [inline src/GoogleAddressInput/google2address.js]: includes does not exist in IE. Use your includes function.

File: src/MultiSelect/MultiSelect.js [hunk 3]
```diff
@@ -106,9 +105,6 @@ class MultiSelect extends InputWithOptions {
       this.props.onSelect(this.optionToTag(option));
     }
 
-    const updeatedOptions = this.getUnselectedOptions();
-    remove(updeatedOptions, option);
```
Comment (3LOK) [inline src/MultiSelect/MultiSelect.js]: Hmmm... any reason this has been deleted?

File: src/GoogleAddressInput/google2address.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
 /*eslint camelcase: off*/
-import includes from 'lodash/includes';
-import omitBy from 'lodash/omitBy';
-import isUndefined from 'lodash/isUndefined';
+import isundefined from 'lodash/isUndefined';
```
Comment (3LOK) [inline src/GoogleAddressInput/google2address.js]: Suggestion: keep the function named like the import (isundefined vs isUndefined)

Comment (Faradey27) [inline src/GoogleAddressInput/google2address.js]: make sense

File: src/GoogleAddressInput/google2address.js (hunk 1 above)
Comment (Faradey27) [inline src/GoogleAddressInput/google2address.js]: it is the same https://jsfiddle.net/hcrj06aw/

Comment (Faradey27) [inline src/GoogleAddressInput/google2address.js]: !!null it is completly same as Boolean(null), which js will do automaticaly

Comment (Faradey27) [inline src/GoogleAddressInput/google2address.js]: @3LOK

File: src/GoogleAddressInput/google2address.js (hunk 2 above)
Comment (Faradey27) [inline src/GoogleAddressInput/google2address.js]: created function and forgot to update this))

File: src/MultiSelect/MultiSelect.js (hunk 3 above)
Comment (Faradey27) [inline src/MultiSelect/MultiSelect.js]: looks like dead code

**Comment by Faradey27** [comment]
@3LOK pull request updated

**Comment by nirhart** [comment]
@3LOK is it still not approved by you?

File: src/MultiSelect/MultiSelect.js (hunk 3 above)
Comment (3LOK) [inline src/MultiSelect/MultiSelect.js]: Right. Indeed seem this way. Weird. @Faradey27

**Comment by 3LOK** [comment]
Sorry. All Good.
@nirhart @Faradey27

**Pull request merged**